docs/integrations/destinations/mysql.md

```diff
@@ -36,6 +36,11 @@ To use the MySQL destination, you'll need:
 * To sync data to MySQL **with** normalization MySQL database 8.0.0 or above
 * To sync data to MySQL **without** normalization you'll need MySQL 5.0 or above.
 
+#### Troubleshooting
+
+Some users reported they could not connect to Amazon AWS RDS MySQL or MariaDB. When creating the new connector they receive a `Cannot create a PoolableConnectionFactory`.
+To solve this issue add `enabledTLSProtocols=TLSv1.2` in the JDBC parameters.
```

Comment:
```suggestion
Some users reported that they could not connect to Amazon RDS MySQL or MariaDB. This can be diagnosed with the error message: `Cannot create a PoolableConnectionFactory`.
To solve this issue add `enabledTLSProtocols=TLSv1.2` in the JDBC parameters.
```